js/languages.js

```diff
@@ -137,6 +137,7 @@
     langRedirector: function (selectedLang) {
       var matchesLang,
         href = document.location.pathname,
```

Comment:
Don't call this `href` if it's actually `pathname`, since that leads to confusion like the bug we are dealing with here.